### PR TITLE
Add library for pdf.js

### PR DESCRIPTION
**Pull request by noahwsmith**
This addresses #8 and should fix the missing PDF.js library on the sandbox. That library must be included separately with Composer, as it turns out.

### CODE REVIEW

**Comment by noahwsmith** [comment]
cc @rosiel - this should address the issues with PDF.js that you found in the sandbox during your demo

File: composer.lock [hunk 1]
```diff
@@ -9362,25 +9502,29 @@
...
                 "drupal"
             ],
+            "support": {
+                "issues": "https://github.com/Islandora/documentation/issues",
+                "source": "https://github.com/Islandora/controlled_access_terms/tree/2.1.1"
+            },
             "time": "2022-01-26T13:15:07+00:00"
         },
         {
             "name": "islandora/crayfish-commons",
```
Comment (nigelgbanks) [inline composer.lock]: Odd for this to be in an install for the Drupal site.

**Review by nigelgbanks — APPROVED** [review]
Looks like a `composer update` was run as well updating other packages. I assume that's intentional and this has been tested. Looks good to me.

**Comment by noahwsmith** [comment]
Thanks Nigel! cc @Natkeeran @rosiel the next build of the sandbox SHOULD include this fix.

**Pull request merged**